logger.pyw

```diff
@@ -87,14 +93,20 @@ def on_press(key):
 
     # keybinds that should be ignored when recording is off go here
 
+    # SUGGESTION: Ignore keys that are longer than one character long that aren't space (ex: alt)
+
     global previous_time
     current_time = time.time()
     delta = int((current_time - previous_time)*1000)
     process = get_active_process()
 
     name = "{current_time} {delta} {key} {process}".format(current_time=current_time,delta=delta,key=str(key), process=process)
     filename = log_dir + name + ".jpg"
-    logging.info(name)
+
+    # Only log name to file if there are 4 columns. Otherwise it will break the stats tool if an error occurs
```

Comment:
I like the idea. Did this create a problem for you? I don't think checking for the 4 columns is the way to go.

I'd prefer if we sanitized the input by replacing " " with "_" for key and process.

**Favorite:** use ; as a delimiter and quote the process name!

Reply:
I think you're thinking unescaped delimiters, which was not the issue I ran into. I ran into a problem where `name` was set to an error, like `NameError: should log was referred to before it was initialized...stack trace line...`. rather than the whole `time delta key application` thing.

I think spaces go nice here because they don't actually appear in any character (as far as I am aware). Although, you might wanna consider using the CSV library and then setting delimiters up as being one of those characters. Additionally, changing this would break all of the data you already have, as you'd have to use sep=':' instead. We could replace all of your spaces with `:` pretty easily, but it still wouldn't fix the issue I had.

Reply:
I can change it if you want though, I just want to make sure you understand the pros and cons of that.

Reply:
Thanks. Let's change to `;` as delimiter AND prevent logging exceptions as Name